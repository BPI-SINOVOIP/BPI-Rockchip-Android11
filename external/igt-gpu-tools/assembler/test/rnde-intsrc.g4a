/* Non-float types are not permitted in the sources of round instructions. */
rnde (1) g0<1>UD g1<0,1,0>UD { align1 };
