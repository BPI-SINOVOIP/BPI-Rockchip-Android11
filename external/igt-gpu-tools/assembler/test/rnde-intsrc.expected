   { 0x00000046, 0x20000021, 0x00000020, 0x00000000 },
